modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,50 @@
+# Imaging Browser
```

Comment:
👎

Reply:
lol

Reply:
hey you said to copy it and only change the content of the different sections !! yoiu didnt say anything of the heading